Stopped browsers anti-aliasing of gender icons

marnovandermaas opened the pull request:
I noticed that with quite a few gender icons, on the borders you could see leakage from adjacent tiles in [`tiles.png`](https://play.idle.land/assets/tiles.png). This is due to the anti-aliasing features of modern browsers. I edited the CSS to stop this anti-aliasing and have tested that this is fixed for Firefox and Chromium.

This pull request also has a small change to the README and the removal of the `package-lock.json` file because this is automatically generated by `npm`.

marnovandermaas committed:
Minor change to README

marnovandermaas committed:
Removed generated package-lock file from version control

marnovandermaas committed:
Stopped anti-aliasing on gender icons

seiyria commented:
Please don't remove the package lock as this locks the dependencies to certain versions. I'll be happy to review the pr at that time. This has been a bit of a nuisance over the years!

marnovandermaas committed:
Revert "Removed generated package-lock file from version control"

This reverts commit 6a8fc8335cd29a0447a5c8cd2beb321e132f4cbb.

As per Seiyria's request put package lock back under version control.

marnovandermaas commented:
Ok, I've reverted that commit. The reason I removed it was that package-lock.json gets completely rewritten when I run `npm install`, but that might be something specific to my setup.

seiyria commented:
What likely happened is that I'm using package file v1 (probably) and yours is v2. That's fine.
